.github/workflows/release-containers.yaml

```diff
@@ -4,10 +4,15 @@ on:
     types:
       - "published"
 
+permissions:
+  packages: write
+
 jobs:
-  publish-container-images:
-    name: publish-container-images
+  build-container-images:
```

Comment:
Could we revert to the previous name of publish-container-images please, and use that for the action too? The important outcome is that it publishes images, so I want that to be immediately obvious. And as a pedantic nit it builds images, not containers.

Reply:
That was left over form my wrong approach before. Changed back to the original name :)